kratos/tests/cpp_tests/strategies/strategies/test_explicit_strategies.cpp

```diff
@@ -24,132 +27,288 @@
+    }
+
+};
+
+
+void GenerateTestExplicitStrategiesModelPart(ModelPart& rModelPart)
+{
+    // Model part settings
+    rModelPart.SetBufferSize(2);
+    rModelPart.AddNodalSolutionStepVariable(TEMPERATURE);
+    rModelPart.AddNodalSolutionStepVariable(REACTION_FLUX);
+    rModelPart.GetNodalSolutionStepVariablesList().AddDof(&TEMPERATURE, &REACTION_FLUX);
+
+    // Create the auxiliary element geometry
+    auto p_node = rModelPart.CreateNewNode(1, 0.0, 0.0, 0.0);
+    rModelPart.AddNode(p_node);
+    p_node->AddDof(TEMPERATURE, REACTION_FLUX);
+
+    // Create and add the auxiliary test element
+    typename GeometryType::PointsArrayType points_vect;
+    points_vect.push_back(p_node);
+    auto p_geom = Kratos::make_shared<Point3D<NodeType>>(p_node);
+    auto p_elem = Kratos::make_intrusive<AuxiliaryExplicitStrategiesTestElement>(1, p_geom);
+    rModelPart.AddElement(p_elem);
+}
+
+template<typename TStrategyType>
+void RunTest(const double tolerance)
+{
+    KRATOS_TRY
+
+    Model current_model;
+    ModelPart& r_model_part = current_model.CreateModelPart("TestModelPart");
+
+    // Set the test model part
+    constexpr double delta_time = 0.01; // Set time step. Must be small to preserve stability for low order methods
+    GenerateTestExplicitStrategiesModelPart(r_model_part); // Create the geometry
+
+    // Create the explicit strategy
+    const bool move_mesh_flag = false;
+    const unsigned int rebuild_level = 0;
+    auto p_explicit_bs = Kratos::make_shared<ExplicitBuilderType>();
+    auto p_explicit_strategy = Kratos::make_unique<TStrategyType>(
+        r_model_part,
+        p_explicit_bs,
+        move_mesh_flag,
+        rebuild_level);
+
+    // Solve and check the two first RK4 steps
+    p_explicit_strategy->Initialize();
+    auto p_test_node = r_model_part.pGetNode(1);
+    p_test_node->FastGetSolutionStepValue(TEMPERATURE) = 50.0; // Set initial solution
+
+    // 1st step
+    r_model_part.CloneTimeStep(delta_time);
+    p_explicit_strategy->InitializeSolutionStep();
+    p_explicit_strategy->SolveSolutionStep();
+    p_explicit_strategy->FinalizeSolutionStep();
+
+    double analytical_1 = (37.5 / 3.5) + (50 - 37.5/3.5) * std::exp(- 3.5 * delta_time);
+    KRATOS_CHECK_NEAR(p_test_node->FastGetSolutionStepValue(TEMPERATURE), analytical_1, tolerance);
+    // 2nd step
+    r_model_part.CloneTimeStep(2.0 * delta_time);
+    p_explicit_strategy->InitializeSolutionStep();
+    p_explicit_strategy->SolveSolutionStep();
+    p_explicit_strategy->FinalizeSolutionStep();
+
+    double analytical_2 = (37.5 / 3.5) + (50 - 37.5/3.5) * std::exp(- 3.5 * 2.0 * delta_time);
+    KRATOS_CHECK_NEAR(p_test_node->FastGetSolutionStepValue(TEMPERATURE), analytical_2, tolerance);
+
+    KRATOS_CATCH("");
+}
+
+template<typename TStrategyType>
+void Solve(ModelPart& rModelPart,
+           Node<3>& rTestNode,
+           const double time,
+           const unsigned int n_steps)
```

Comment:
```suggestion
void Solve(
    ModelPart& rModelPart,
    Node<3>& rTestNode,
    const double time,
    const unsigned int n_steps)
```
(minor style guide suggestion)